Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -154,12 +155,18 @@ open class ReadingThemesControlsViewController: UIViewController {
             fallthrough
         case Theme.dark.name:
             imageDimmingSwitch.isEnabled = true
+            makeImageDimmingSwitchMoreProminentForDisabledState(imageDimmingSwitch.isEnabled)
             applyBorder(to: darkThemeButton)
         default:
             break
         }
     }
     
+    func makeImageDimmingSwitchMoreProminentForDisabledState(_ isEnabled: Bool) {
```

Comment:
If you're going to change this at all, it should be combined with setting `isEnabled` on the switch:

`var isImageDimmingSwitchEnabled { didSet { ... } }`